src/bee.ts

```diff
@@ -297,6 +299,66 @@ export class Bee {
     return chunk.download(this.getRequestOptionsForCall(options), reference)
   }
 
+  /**
+   * Adds grantees to a postage batch.
```

Comment:
The description is not accurate. addGrantees creates a list of grantees

Reply:
fixed https://github.com/Solar-Punk-Ltd/bee-js/commit/a2dc53c04ef239ac3632d54b32f0cfa91456528e

Reply:
I meant that the "creates for a postage batch" is incorrect, instead something like: "it creates a list from the given array of public keys"

Reply:
fixed https://github.com/Solar-Punk-Ltd/bee-js/commit/d739873ed4e022c01d5c6e796bcac6914924ac2a